### DISCUSSION TITLE
como deixar o site responsivo?????

### DISCUSSION THREAD

**Discussion by KaliKali90** [frontendbr/forum#2491] [Perguntas e Respostas]
criei um site para um trabalho da faculdade, porém o posicionamento das divs estão muito bagunçada e são visiveis apenas em desktop. como faço pra deixar todo em ordem sem alterar o conteudo? existe alguma maneira de ajustar tudo de uma vez só? (HTML e CSS Apenas)

**Comment by devpro999** [comment]
xiiiiiiiiii

**Reply by KaliKali90** [reply]
eu sei quem você é, e devia fazer sua parte com ela. Ela diz que isso é infantil

**Comment by felquis** [comment]
Não existe um jeito de deixar tudo responsivo de uma vez só, no CSS são algumas práticas que permitem que isso seja possível:

- Usar a técnica do `rem` para o tamanho das fontes
- evitar usar width e height com números fixos ex: 100px, sempre usar 100%, auto, flex, grid
- usar flex e grid pois com eles é mais fácil reordenar columns e rows de acordo com media queries
- media queries e container queries
- usar técnicas de mobile first, em vez de fazer responsivo para mobile, faz responsivo para desktop

Essas são de forma bem geral, o que eu faço e parece ser o mais normal
